code/game/gamemodes/miniantags/abduction/gland.dm

```diff
@@ -96,15 +149,25 @@
 	for(var/mob/living/carbon/H in orange(4,T))
 		if(H == owner)
 			continue
-		to_chat(H, "<span class='alien'>You hear a buzz in your head.</span>")
-		H.AdjustConfused(20)
+		switch(pick(1,3))
+			if(1)
+				to_chat(H, "<span class='userdanger'>You hear a loud buzz in your head, silencing your thoughts!</span>")
+				H.Stun(3)
+			if(2)
+				to_chat(H, "<span class='warning'>You hear an annoying buzz in your head.</span>")
+				H.AdjustConfused(15)
+				H.adjustBrainLoss(5, 15)
+			if(3)
+				H.hallucination += 60
 
 /obj/item/organ/internal/heart/gland/pop
 	cooldown_low = 900
 	cooldown_high = 1800
 	uses = -1
-	human_only = 1
+	human_only = TRUE
 	icon_state = "species"
+	mind_control_uses = 5
+	mind_control_duration = 300
```

Comment:
Are you sure you meant "300" here? It seems uncharacteristically low compared to the rest of the glands